doc/man3/ASYNC_start_job.pod

```diff
@@ -363,6 +363,8 @@ ASYNC_init_thread, ASYNC_cleanup_thread,
 ASYNC_start_job, ASYNC_pause_job, ASYNC_get_current_job, ASYNC_get_wait_ctx(),
 ASYNC_block_pause(), ASYNC_unblock_pause() and ASYNC_is_capable() were first
 added in OpenSSL 1.1.0.
+ASYNC_set_mem_functions(), ASYNC_get_mem_functions() were added
+in OpenSSL 3.2.0.
```

Comment:
We are inconsistent about whether we include the patch number or not in the version (i.e. 3.2 or 3.2.0). I think in general we should be using the number *without* the patch, i.e. 3.2. The only time I think we should have an explicit patch number is if we have introduced a function after the initial ".0" release (which is rare).

Reply:
If this is a general statement about the documentation, then that is true (just checked `grep -r "3\.2\.0" doc/ | wc` getting 32 hits). For now the latest commit removed all "3.x.0" instances (8 of the 32 in this example) introduced by this PR _not_ touching the other instances. Please let me know if this is not what you'd expect @mattcaswell .